Functions/FirebaseFunctions/Public/FirebaseFunctions/FIRFunctions.h

```diff
@@ -44,15 +44,31 @@ NS_SWIFT_NAME(Functions)
  */
 + (instancetype)functionsForRegion:(NSString *)region NS_SWIFT_NAME(functions(region:));
 
+/**
+ * Creates a Cloud Functions client with the default app and given custom domain.
+ * @param customDomain A custom domain for the http trigger, such as "https://mydomain.com".
+ */
+// clang-format off
```

Comment:
Would be nice to verify if this is still really needed, but perhaps ok to deal with separately along with the other existing disables? @ryanwilson any thoughts here.

Reply:
I think it is.  I took it out and the formatter moved by `SWIFT_NAME` thing